### ISSUE TITLE
Profile defined in `light-client-js` is not respected

### ISSUE THREAD

**Issue by xla** [informalsystems/tendermint-rs#887]
Cargo does not respect `profile` directives in crate cargos and instead requires it to be done in the workspace, currently this manifests with:

```
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /home/xla/dev/src/github.com/informalsystems/tendermint-rs/light-client-js/Cargo.toml
workspace: /home/xla/dev/src/github.com/informalsystems/tendermint-rs/Cargo.toml
    Finished dev [unoptimized + debuginfo] target(s) in 0.07
```

This originates from
https://github.com/informalsystems/tendermint-rs/blob/74ea770084c7bdddfe83265b055f7078e5046d0d/light-client-js/Cargo.toml#L49-L51

From an initial discussion with @romac this can either be removed and left to the downstream consumers of the crate, or a different method could be devised to enforce smaller code size for this specific create. Any input on this? @ethanfrey @webmaster128

**Comment by ethanfrey** [comment]
We do this in the top level (workspace) cargo.toml for cosmwasm-plus

https://github.com/CosmWasm/cosmwasm-plus/blob/main/Cargo.toml#L56

Maybe this is what you are looking for.

**Comment by thanethomson** [comment]
Closed by #936
